esm/pretrained.py

```diff
@@ -267,3 +293,14 @@ def esm1v_t33_650M_UR90S_5():
     Returns a tuple of (Model, Alphabet).
     """
     return load_model_and_alphabet_hub("esm1v_t33_650M_UR90S_5")
+
+
+def esm_if1_gvp4_t16_142M_UR50():
```

Comment:
we have to add this function also to `hubconf.py`

Reply:
Added